lib/tasks/learn_to_rank.rake

```diff
@@ -15,7 +25,7 @@ namespace :learn_to_rank do
 
     data = LearnToRank::DataPipeline::Bigquery.fetch(JSON.parse(Base64.decode64(ENV["BIGQUERY_CREDENTIALS"])), viewcount: args.viewcount || 10)
 
-    CSV.open("tmp/#{args[:output]}.csv", "wb") do |csv|
+    csv_body = CSV.generate(headers: true) do |csv|
```

Comment:
What sort of size are we talking for this file? Asking cos I think this is changing it from streaming to buffering the whole thing in RAM. (If it's like tens of megabytes then no big deal. If hundreds then might be worth a few mins faff to keep it streaming?)

Reply:
Good point - largest file is 50mb, so will leave for now.